Makefile

```diff
@@ -20,11 +20,11 @@
 #==================================================================================================
 
 
-CPP                =
+CPP                = 
 PYTHON             = python
 COMPILER_MODE      = DEBUG
 PRECISION          = DOUBLE
-OPENMP             = 1
+OPENMP             = 0
```

Comment:
Yes, I agree.  Also, shouldn't the default for COMPILER_MODE be FAST rather than DEBUG since that'll be the default/most common mode when people want to download and use the code?  (DEBUG hopefully only being used rarely for obvious reasons).